src/lexer.rs

```diff
@@ -164,12 +231,46 @@ impl JavascriptLexer {
           '{' => {
             pending_import.expected_token = ImportToken::NamedImport;
           },
+          '\u{0041}'..='\u{2FA1D}' => {
+            pending_import.expected_token = ImportToken::DefaultImport;
```

Comment:
I don't think the logic here accounts for whether we've already found the default or named exports section. For example, something like this is invalid, but i think that the code here would be fine with it:

```js
import default1, { named1 }, default 2 from './thing.js';
```

In other words, the checks to set the expected_token to DefaultImport or NamedImports, should first verify whether we have already found a default import or named imports.

Reply:
This is a great point.  I would love to add this once we have a test suite spun up.  Or perhaps do you have some ideas on what we can throw in here for now?

Reply:
Sure, I'm happy to keep this as a to-do in a future PR.

Reply:
I just realized.  The default import has to be first.  So we can check the pending import and see if the `JavascriptImport` has named imports already.  If not, then we can parse!